internal/controller/release_controller.go

```diff
@@ -389,10 +389,7 @@ func (r *ReleaseReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	if err != nil {
 		return err
 	}
-	//
-	if !r.CreateManagement {
-		return nil
-	}
```

Comment:
i'd suggest not to create release if both `createManagement` and `createRelease` are turned off

Reply:
Ok I've changed that to be if both are false then return nil instead